client/src/frontend/containers/ReadingGroup/index.js

```diff
@@ -1,134 +1,70 @@
-import React, { Component } from "react";
-import get from "lodash/get";
-import { readingGroupsAPI, requests } from "api";
+import React, { useState } from "react";
+import { readingGroupsAPI } from "api";
+import { useParams, useHistory, useLocation } from "react-router-dom";
 import { childRoutes } from "helpers/router";
+import { useDispatch } from "react-redux";
 import lh from "helpers/linkHandler";
-import connectAndFetch from "utils/connectAndFetch";
-import { grab } from "utils/entityUtils";
-import { entityStoreActions } from "actions";
 import HeadContent from "global/components/HeadContent";
 import Drawer from "global/containers/drawer";
 import { RegisterBreadcrumbs } from "global/components/atomic/Breadcrumbs";
 import { GroupHeading } from "frontend/components/reading-group/headings";
 import Settings from "frontend/components/reading-group/Settings";
 import SearchDialog from "frontend/components/collecting/SearchDialog";
+import { useFetch, useFromStore } from "hooks";
 
-const { request } = entityStoreActions;
+export default function ReadingGroup({ route }) {
```

Comment:
There's definitely more refactoring that can be done to the child containers of this one. I tried to only do what was needed to replace the hooks, but I'd appreciate input on whether to keep going.